target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -933,7 +1024,8 @@ TARGET_DEVICES += AGPF-S0
 ### Sagem ###
 define Device/FAST2404
   $(Device/bcm63xx)
-  DEVICE_TITLE := Sagem F@ST2404
+  DEVICE_VENDOR := Sagem
+  DEVICE_MODEL := F@ST2404
```

Comment:
Might as well fix the device names, Sagem(com) doesn't seem to use all caps, but rather e.g. "F@st 2404".

Reply:
done

Reply:
I also changed Sagem to Sagemcom. The device are likely produced by Sagemcom after they split off Sagem in 2009.